README.md

```diff
@@ -1771,6 +1771,22 @@ end
 
 The `rescue_from` block must return a `Rack::Response` object, call `error!` or re-raise an exception.
 
+The `with` keyword is available as `rescue_from` options, it can be passed method name or `Rack::Response` object.
+
+```ruby
+class Twitter::API < Grape::API
+  format :json
+  helpers do
+    def server_error!
+      error!({ error: 'Server error.' }, 500, { 'Content-Type' => 'text/error' })
+    end
+  end
+
+  rescue_from :all,          with: :server_error!
+  rescue_from ArgumentError, with: Rack::Response.new('rescued with a method', 400)
```

Comment:
Yeah, you're right. But the dangerous behaviour is [expected](https://github.com/ruby-grape/grape/blob/master/spec/grape/api_spec.rb#L1506-L1519).
Should I remove that and introduce lambda approach?